Gemfile

```diff
@@ -25,7 +25,7 @@ group :test, :development do
 	  gem lib, git: "git://github.com/rspec/#{lib}.git"
 	end
   gem "shoulda-matchers"
-  gem "database_cleaner", "1.5.3"
```

Comment:
sadly, there's a bug in 1.5.3 that won't work with our version of mongo 😞 I opened a pr in the project but it hasn't been super active so I'm not sure if it'll make it in there